lib/dgram.js

```diff
@@ -495,6 +495,19 @@ Socket.prototype.setMulticastLoopback = function(arg) {
 };
 
 
+Socket.prototype.setMulticastInterface = function(interfaceAddress) {
+  this._healthCheck();
+
+  if (!interfaceAddress) {
```

Comment:
either that or `interfaceAddress` should be explicitly coerced, e.g. `interfaceAddress = String(interfaceAddress)`